chromium_src/components/sync_device_info/device_info_sync_bridge.cc

```diff
@@ -3,13 +3,18 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
+#include "components/sync_device_info/device_info_sync_bridge.h"
 #include "brave/components/sync_device_info/brave_device_info.h"
```

Comment:
a new line would be nice

Reply:
hmm. this should be a first line: ```#include "components/sync_device_info/device_info_sync_bridge.h"```, then a new line and then ```brave/...``` include.

Reply:
sorry, made a mistake. fixed now